templates/news/news-doc-en.html

```diff
@@ -0,0 +1,87 @@
+---
+title: Documentation about News template
+description: Documentation and working examples for the News template.
+language: en
+altLangPage: news-doc-fr.html
+dateModified: 2023-02-21
+share: true
+---
+
+<div class="wb-prettify all-pre hide"></div>
+
+<dl class="horizontal">
+	<dt>Status</dt>
+	<dd>Stable</dd>
+	<dt>Type</dt>
+	<dd>Canada.ca template</dd>
+	<dt>Last review</dt>
+	<dd>2019-01-30</dd>
+	<dt>Conforming to</dt>
+	<dd>Content and IA spec 2.1</dd>
+	<dt>Specification</dt>
+	<dd><a href="https://design.canada.ca/mandatory-templates/news-pages.html#landing">News pages - Canada.ca mandatory template</a></dd>
+	<dt>Template version</dt>
+	<dd>Version 2.0</dd>
+</dl>
+
+<h2>Working examples</h2>
+<ul>
+	<li><a href="news-en.html">Working example</a></li>
+	<li><a href="news-fr.html" lang="fr">Exemple pratique</a></li>
+</ul>
+
+<h2>Deprecated layout</h2>
+<p>As of version 5.0.0 of GCWeb, the promotions use version 5.0 of Context-specific features. The following code sample is therefore deprecated.</p>
+<pre><code>&lt;aside class="prm-flpr"&gt;
+	&lt;div class="container"&gt;
+		&lt;h2 class="wb-inv"&gt;Promotions&lt;/h2&gt;
+		&lt;div class="wb-tabs carousel-s2 show-thumbs playing slow"&gt;
+			&lt;ul role="tablist"&gt;
+				&lt;li class="active"&gt;
+					&lt;a href="#tab1"&gt;&lt;img src="./img/promos/1170x347-1.png" alt="&gt;&lt;span class="wb-inv"&gt;Tab 1: Banff National Park&lt;/span&gt;&lt;/a&gt;
+				&lt;/li&gt;
+				&lt;li&gt;
+					&lt;a href="#tab2"&gt;&lt;img src="./img/promos/1170x347-2.png" alt="&gt;&lt;span class="wb-inv"&gt;Tab 2: Algonquin Provincial Park&lt;/span&gt;&lt;/a&gt;
+				&lt;/li&gt;
+				&lt;li&gt;
+					&lt;a href="#tab3"&gt;&lt;img src="./img/promos/1170x347-3.png" alt="&gt;&lt;span class="wb-inv"&gt;Tab 3: Rideau Canal&lt;/span&gt;&lt;/a&gt;
+				&lt;/li&gt;
+			&lt;/ul&gt;
+			&lt;div class="tabpanels"&gt;
+				&lt;div role="tabpanel" id="tab1" class="in fade"&gt;
+					&lt;a href="#" class="learnmore"&gt;
+						&lt;figure&gt;
+							&lt;img src="./img/promos/1170x347-1.png" alt="&gt;
+							&lt;figcaption&gt;
+								&lt;p&gt;Banff National Park &lt;/p&gt;
+							&lt;/figcaption&gt;
+						&lt;/figure&gt;
+					&lt;/a&gt;
+				&lt;/div&gt;
+				&lt;div role="tabpanel" id="tab2" class="out fade"&gt;
+					&lt;a href="#" class="learnmore"&gt;
+						&lt;figure&gt;
+							&lt;img src="./img/promos/1170x347-2.png" alt="&gt;
+							&lt;figcaption&gt;
+								&lt;p&gt;Algonquin Provincial Park &lt;/p&gt;
+							&lt;/figcaption&gt;
+						&lt;/figure&gt;
+					&lt;/a&gt;
+				&lt;/div&gt;
+				&lt;div role="tabpanel" id="tab3" class="out fade"&gt;
+					&lt;a href="#" class="learnmore"&gt;
+						&lt;figure&gt;
+							&lt;img src="./img/promos/1170x347-3.png" alt="&gt;
+							&lt;figcaption&gt;
+								&lt;p&gt;Rideau Canal &lt;/p&gt;
+							&lt;/figcaption&gt;
+						&lt;/figure&gt;
+					&lt;/a&gt;
+				&lt;/div&gt;
+			&lt;/div&gt;
+		&lt;/div&gt;
+	&lt;/div&gt;
+&lt;/aside&gt;</code></pre>
```

Comment:
Ajoute une référence à l'élément suivant.

```suggestion
&lt;/aside&gt;
&lt;main>
   ...
&lt;/main></code></pre>
```